src/client/containers/dashboard-page/dashboard-page.component.js

```diff
@@ -54,6 +58,12 @@ const DashboardPageContainer = () => {
       },
     ).then((data) => data.json());
     setBoundaryData(defaultValues);
+
+    const statusData = await fetch('/api/batch-status/1', {
```

Comment:
How does the batch-status endpoint work? I'm just wondering if the `1` is a hardcoded value?

Reply:
it is hardcoded, since we are showing data only for batch with id 1, in PR #209 will be not harcoded